flowsa/methods/flowbysectormethods/GHG_national_m2_common.yaml

```diff
@@ -60,18 +60,22 @@ attribution_sources:
 _sources_to_cache:
   Detail_Use_YEAR:  ## Update year
     !include:Detail_Use_SUT.yaml
+    industry_spec: !include:BEA_detail_target.yaml:industry_spec
```

Comment:
@bl-young This industry_spec isn't necessary now because it is the exact info pulled from Detail_Use_SUT.yaml, so I think these 4 lines should be deleted

Reply:
right. ok